tools/tracer/README

```diff
@@ -0,0 +1,60 @@
+THIS DOCUMENT CONTAINS INFORMATION ON PRODUCTS IN THE DESIGN PHASE OF DEVELOPMENT.
+
+Copyright(c) 2014-2020 Intel Corporation. All Rights Reserved.
+
+
+Overview
+========
+
+Media SDK Tracer is a tool which permits to dump logging information from the calls
+of the application to the Media SDK library. Trace log obtained from this tool is a
+recommended information to provide to Media SDK team on submitting questions and
+issues.
+
+Media SDK tracer tool version is: v1.32.0. Note that 1.32 stands for the Media SDK API
+version this particular version of the Tracer is intended for.
+
+
+Prerequisites
+=============
+
+To use this version of the tracer, it is required to have the following products installed
+on the system:
+- Intel Media Stack: https://github.com/Intel-Media-SDK/MediaSDK/releases
```

Comment:
This is built along with the mediasdk tree, not stand alone. As of now this does not make sense.... I.e. remove prerequisites.

Reply:
done